### PR TITLE
Add other architectures and update to 2.0.1.post1

### PR DESCRIPTION
**Pull request by pavelzw**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by pavelzw** [comment]
@conda-forge-admin, please rerender

**Comment by pavelzw** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,17 @@
 {% set name = "unicorn" %}
-{% set version = "1.0.3" %}
+{% set version = "2.0.1.post1" %}
 
 package:
   name: {{ name|lower }}
```
Comment (xhochy) [inline recipe/meta.yaml:5]: Let's start with this
```suggestion
  name: {{ name|lower }}-split
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -39,7 +39,6 @@ outputs:
       script:
         - export LIBUNICORN_PATH="${PREFIX}"
         - mkdir source/bindings/python/src
-        - cp source/pkgconfig.mk source/bindings/python/src
         - ${PYTHON} -m pip install source/bindings/python/ -vv
     requirements:
       host:
```
Comment (xhochy) [inline recipe/meta.yaml:44]: Please use  `- {{ pin_subpackage("unicorn", exact=True) }}` here in `host` and build`

File: recipe/meta.yaml [hunk 3]
```diff
@@ -86,3 +85,4 @@ about:
 extra:
```
Comment (xhochy) [inline recipe/meta.yaml:85]:
```suggestion
extra:
  feedstock-name: unicorn
```

File: recipe/build.sh [hunk 4]
```diff
@@ -1,10 +1,14 @@
 #!/usr/bin/env bash
 
-mkdir build && cd build
+cd source
 
-cmake ${CMAKE_ARGS} -LAH \
+mkdir build
+
+cmake -B build \
+    ${CMAKE_ARGS} -LAH \
```
Comment (xhochy) [inline recipe/build.sh:8]:
```suggestion
    ${CMAKE_ARGS} -LAH \
    -DCMAKE_C_COMPILER=${CC} \
    -DCMAKE_CXX_COMPILER=${CXX} \
```

File: recipe/build.sh [hunk 5]
```diff
@@ -1,10 +1,14 @@
 #!/usr/bin/env bash
 
-mkdir build && cd build
+cd source
 
-cmake ${CMAKE_ARGS} -LAH \
+mkdir build
```
Comment (xhochy) [inline recipe/build.sh:5]:
```suggestion
mkdir build

if [[ "${target_platform}" == linux-* ]]; then
  export LDFLAGS="-lrt ${LDFLAGS}"
fi
```

File: recipe/build.sh [hunk 6]
```diff
@@ -1,10 +1,14 @@
...
+
+cmake -B build \
+    ${CMAKE_ARGS} -LAH \
+    -DCMAKE_C_FLAGS=-lrt \
```
Comment (xhochy) [inline recipe/build.sh:9]:
```suggestion
```

File: recipe/build.sh [hunk 7]
```diff
@@ -1,10 +1,14 @@
...
+    -DCMAKE_BUILD_TYPE=Release \
     -DCMAKE_INSTALL_LIBDIR=lib \
```
Comment (xhochy) [inline recipe/build.sh:11]: Part of `CMAKE_ARGS`
```suggestion
```

**Comment by pavelzw** [comment]
@conda-forge-admin, please rerender

**Comment by pavelzw** [comment]
@conda-forge-admin, please rerender

**Comment by pavelzw** [comment]
@conda-forge-admin, please rerender

**Comment by 0xbe7a** [comment]
@conda-forge-admin, please rerender

**Comment by pavelzw** [comment]
Closes #7 
Closes #8 
Closes #9 
Closes #10

File: recipe/meta.yaml [hunk 8]
```diff
@@ -1,53 +1,65 @@
...
+        - ${PYTHON} -m pip install . -vv
     requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
       host:
+        - {{ pin_subpackage("unicorn", exact=True) }}
         - pip
-        - python >=2.7
+        - python >=3.6
```
Comment (xhochy) [inline recipe/meta.yaml:59]:
```suggestion
        - python
```

File: recipe/meta.yaml [hunk 9]
```diff
@@ -1,53 +1,65 @@
...
       run:
-        - python >=2.7
-        - {{ pin_subpackage("unicorn", max_pin="x.x.x") }}
+        - python >=3.6
```
Comment (xhochy) [inline recipe/meta.yaml:61]:
```suggestion
        - python
```

**Review by xhochy — CHANGES_REQUESTED** [review]
Don't use `python >=` on non-noarch packages.

**Comment by xhochy** [comment]
@conda-forge-admin please rerender

**Review by xhochy — APPROVED** [review]
LGTM, leaving open for @chrisburr to review

**Comment by chrisburr** [comment]
Thanks for updating this and thanks to @xhochy for reviewing! 🍰

**Pull request merged**